elements/pfe-icon/test/index.html

```diff
@@ -12,12 +12,12 @@
     // Load and run all tests (.html, .js):
     WCT.loadSuites([
       'pfe-icon_test.html',
-      'pfe-icon_react_test.html',
-      'pfe-icon_vue_test.html',
+      // 'pfe-icon_react_test.html',
```

Comment:
The other tests suites are still commented out

Reply:
OH CRAP, good catch.

Sorry, I was debugging something else and forgot to take this back 😬